docs(auth-kit): Update docs for the new `SafeAuthPack` (#289)
safe-global/safe-docs: safe-core-sdk/auth-kit/reference/AuthKitBasePack.md

}
```

## Abstract methods and properties that any pack must implement

> These methods and properties are the common interface for all the Auth packs. Check each pack's documentation to get more details.

Reviewer: ```suggestion
These methods and properties are the common interface for all the Auth packs. Check each pack's documentation for more details.
```

### `isAuthenticated(): boolean`

> Any pack extending the `AuthKitBasePack` class must implement the `isAuthenticated()` getter. This method returns a `boolean` indicating if the user is authenticated or not.

Reviewer: ```suggestion
Any pack extending the `AuthKitBasePack` class must implement the `isAuthenticated()` getter. This method returns a `boolean` indicating whether the user is authenticated.
```
